### ISSUE TITLE
Unable to download nwjs-X-osx-x64.zip

### ISSUE THREAD

**Issue by madwax** [nwjs/nw-builder#319]
This is an odd one. I'm trying to use nw-builder or grunt-nw-builder and every time it fails while trying to download the osx-x64 zip file of any version I try.  The problem happens on OSX  10.10.5 and Windows 8.1. I can download the files using the url. 

Output:
Running "nwjs:src" (nwjs) task
Using v0.13.0
Create cache folder in /Users/madwax/devel/nw/example/cache/0.13.0
Downloading: http://dl.nwjs.io/v0.13.0/nwjs-v0.13.0-win-ia32.zip
Create cache folder in /Users/madwax/devel/nw/example/cache/0.13.0
Downloading: http://dl.nwjs.io/v0.13.0/nwjs-v0.13.0-win-x64.zip
Create cache folder in /Users/madwax/devel/nw/example/cache/0.13.0
Downloading: http://dl.nwjs.io/v0.13.0/nwjs-v0.13.0-osx-ia32.zip
Create cache folder in /Users/madwax/devel/nw/example/cache/0.13.0
Downloading: http://dl.nwjs.io/v0.13.0/nwjs-v0.13.0-osx-x64.zip
ERROR: The version 0.13.0 does not have a corresponding build posted at http://dl.nwjs.io/. Please choose a version from that list.
Fatal error: Unable to download nodewebkit.
mue:example madwax$ grunt
Running "nwjs:src" (nwjs) task
Using v0.14.3
Create cache folder in /Users/madwax/devel/nw/example/cache/0.14.3
Downloading: http://dl.nwjs.io/v0.14.3/nwjs-v0.14.3-win-ia32.zip
Create cache folder in /Users/madwax/devel/nw/example/cache/0.14.3
Downloading: http://dl.nwjs.io/v0.14.3/nwjs-v0.14.3-win-x64.zip
Create cache folder in /Users/madwax/devel/nw/example/cache/0.14.3
Downloading: http://dl.nwjs.io/v0.14.3/nwjs-v0.14.3-osx-ia32.zip
Create cache folder in /Users/madwax/devel/nw/example/cache/0.14.3
Downloading: http://dl.nwjs.io/v0.14.3/nwjs-v0.14.3-osx-x64.zip
  downloading [--------------------] 0% 297.8sERROR: The version 0.14.3 does not have a corresponding build posted at http://dl.nwjs.io/. Please choose a version from that list.
Fatal error: Unable to download nodewebkit.

**Comment by craigbloodworth** [comment]
Yep I'm also seeing this problem

**Comment by craigbloodworth** [comment]
Seems that os32 is missing (maybe it has been discontinued?) from recent builds and so it fails looking for http://dl.nwjs.io/v0.14.3/nwjs-v0.14.3-osx-ia32.zip

**Comment by evshiron** [comment]
Well, if you click http://dl.nwjs.io/v0.14.3/nwjs-v0.14.3-osx-ia32.zip, you will end up with a 404. Because `nw-builder` downloads all selected (and default includes `osx32`, which is deprecated) platforms in parallel and once one fails, all other downloads fail as well.

**Comment by madwax** [comment]
Sorry my fault. I didn't see if that the 32bit version that was failing.

**Comment by adam-lynch** [comment]
Closing this in favour of #327. Thanks for reporting.
